api/swagger.yaml

```diff
@@ -8746,6 +8771,7 @@ paths:
           description: "no error"
           schema:
             type: "object"
+            title: "UnlockKey"
```

Comment:
`SwarmUnlockKeyResponse`

Reply:
right. I wonder it would make sense to define this as a global type as same structure is used for /swarm/unlock and /swarm/unlockkey

Reply:
I'm not sure if they will stay the same. Let's keep them separate for now